Sources/CorePayments/CoreConfig.swift

```diff
@@ -5,10 +5,10 @@ import Foundation
 public struct CoreConfig {
 
     public let environment: Environment
-    public let accessToken: String
+    public let clientID: String
 
-    public init(accessToken: String, environment: Environment) {
+    public init(clientID: String, environment: Environment) {
```

Comment:
This needs a changelog since it is a breaking change. We can update similar to the changelog used in the Android PR.

Reply:
Yes, going in the next commit.